php_phongo.c

```diff
@@ -1461,6 +1461,45 @@ static const char* php_phongo_bson_type_to_string(bson_type_t type) /* {{{ */
 		bson_iter_key(&(iter)),                        \
 		php_phongo_bson_type_to_string(bson_iter_type(&(iter))))
 
+static bool php_phongo_uri_finalize_auth(mongoc_uri_t* uri TSRMLS_DC) /* {{{ */
+{
+	const char *source = NULL;
+
+	source = mongoc_uri_get_auth_source(uri);
```

Comment:
Since `source` isn't used elsewhere, I think we can consolidate this declaration and assignment into one line and move it into the second `if` block (after checking for GSSAPI or MONGODB-X509).

```c
const char *source = mongoc_uri_get_auth_source(uri);
```